controllers/alertrulegroup_controller.go

```diff
@@ -118,9 +118,10 @@ func (r *GrafanaAlertRuleGroupReconciler) Reconcile(ctx context.Context, req ctr
 		return ctrl.Result{}, fmt.Errorf("folder uid not found, alert rule must reference a folder")
 	}
 
-	editable := "true" //nolint:goconst
-	if group.Spec.Editable != nil && !*group.Spec.Editable {
-		editable = "false"
+	var disableProvenance *string
+	if group.Spec.Editable == nil || *group.Spec.Editable {
```

Comment:
I think v5.19.0 introduced a couple of bugs related to the `editable` field in alert rules.
- It broke the field entirely (As you already fixed)
- It changed the default to `true` for the same reason

We generally want the default to be `editable: false` when `nil`. This discourages editing in the UI and potentially losing work on the next synchronization.
```suggestion

	if group.Spec.Editable != nil && *group.Spec.Editable {
```

Also, we have a lot of lints, so a whitespace above the `if` is required 🙂